pkg/gen/BUILD.bazel

```diff
@@ -0,0 +1,74 @@
+load(":gen.bzl", "hoist_files", "go_proto_srcs")
+
+hoist_files(
+    name = "go_proto",
+    data = [":go_proto_srcs"],
+)
+
+#  bazel query 'kind(go_proto_library, //pkg/...)' | sort | awk '{ printf("\"%s\",\n", $0) }'
+
+go_proto_srcs(
```

Comment:
What do you have in mind? I've contemplated a few ideas but don't love any.

Reply:
We already autogenerate `pkg/BUILD.bazel` (part of `build/bazelutil/bazel-generate.sh`, via `generate-test-suites`). It just uses `go_test` in the `bazel query` rather than `go_proto_library` as you would do for generating this file.

Reply:
I'll do it in a follow-up